test/production/terser-class-static-blocks/terser-class-static-blocks.test.ts

```diff
@@ -10,4 +12,21 @@ describe('terser-class-static-blocks', () => {
     const $ = await next.render$('/')
     expect($('p').text()).toBe('hello world')
   })
+
+  if (!isNextDeploy) {
+    it('should have stripped away all comments', async () => {
+      const chunksDir = path.join(next.testDir, '.next/static')
+      const chunks = glob.sync('**/*.js', {
+        cwd: chunksDir,
+      })
+
+      expect(chunks.length).toBeGreaterThan(0)
+
+      for (const chunk of chunks) {
+        expect(
+          await next.readFile(path.join('.next/static', chunk))
+        ).not.toContain('/*')
```

Comment:
Could we check in more assuring way than using `.not`?

Q: Would using `Promise.all` have CI perf win or is trivial?

Reply:
Doesn't hurt to `Promise.all()` can update

Reply:
I think we need to use `.not` but expanded to also check for the added comment text as well